README.md

```diff
@@ -1,13 +1,20 @@
 # vitasdk code samples
 
-Every directory contains a single sample. To build all samples, add the toolchain `bin/` directory to your `$PATH`, then run `./build.sh`. To clean all samples, run `./clean.sh`. To build a single sample, `cd` to its directory and type `make`.
+## Prerequists
 
-Always use the `*_debug.bat` (e.g. `run_homebrew_unity_debug.bat`) launcher to get debug output from Vita.
+In order to build a vita sample, you need to add the toolchain `bin/` directory to your `$PATH`.
 
-## List of samples
+## Building
 
-* `hello_world`: A minimal Makefile-based C project
-* `hello_cpp_world`: A minimal Makefile-based C++ project
```

Comment:
Because nobody will remember to update the main README when they will add/change a sample (for example, the net_http sample were already missing ;-) ).

:+1: for the per-sample README

We could just explain in this main README that every samples have they own README
So we don't have to update the main one each time.